debian/marquee/nginx/sites/images.whatwg.org.conf

```diff
@@ -8,9 +8,8 @@ server {
     include /etc/nginx/whatwg.conf;
 
     location / {
-        autoindex on;
-        autoindex_localtime off;
-        autoindex_exact_size off;
+        fancyindex on;
```

Comment:
Isn't this redundant, both before and after this change?

Reply:
>> debian/marquee/nginx/sites/images.whatwg.org.conf
>> ```
>> fancyindex on;
>> ```
>
> Isn't this redundant, both before and after this change?

Maybe. I’ll re-test and see. But IIRC the reason I added it to begin with was to to prevent the way that nginx has of otherwise causing nested directives in those `location` blocks to have the effect of clobbering the settings for other unrelated directives specified higher up.